### PR TITLE
docs: add in mill-github-dependency-graph to third party plugins

### PR DESCRIPTION
**Pull request by ckipp01**

### CODE REVIEW

File: docs/antora/modules/ROOT/pages/Thirdparty_Plugins.adoc [hunk 1]
```diff
@@ -197,6 +197,42 @@ object `jvm-project` extends JavaModule with GitVersionedPublishModule {
...
+Project home: https://github.com/ckipp01/mill-github-dependency-graph
+
+=== Quickstart
+
+The easiest way to use this plugin is with the
+https://github.com/ckipp01/mill-github-dependency-graph[mill-dependency-submission]
+action. You can add it as a workflow:
+
+[source,yaml]
+---
```
Comment (lefou) [inline docs/antora/modules/ROOT/pages/Thirdparty_Plugins.adoc:215]:
```suggestion
----
```

File: docs/antora/modules/ROOT/pages/Thirdparty_Plugins.adoc [hunk 2]
```diff
@@ -197,6 +197,42 @@ object `jvm-project` extends JavaModule with GitVersionedPublishModule {
...
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: coursier/cache-action@v6
+    - uses: actions/setup-java@v3
+      with:
+        distribution: 'temurin'
+        java-version: '17'
+    - uses: ckipp01/mill-dependency-submission@v1
+---
```
Comment (lefou) [inline docs/antora/modules/ROOT/pages/Thirdparty_Plugins.adoc:234]:
```suggestion
----
```

**Review by lefou — COMMENTED** [review]
Nice. A small issue with the formatting of the code block. I added suggestions for a fix. 

Also, the quickstart example uses more than just the mentioned plugin but also the `mill-dependency-submission` action. Maybe, this is worth mentioning.

**Comment by ckipp01** [comment]
> Also, the quickstart example uses more than just the mentioned plugin but also the mill-dependency-submission action. Maybe, this is worth mentioning?

Yea, I actually have it mentioned right above the code snippet that the easiest way to use the plugin is via that action.

**Comment by lefou** [comment]
> > Also, the quickstart example uses more than just the mentioned plugin but also the mill-dependency-submission action. Maybe, this is worth mentioning?
> 
> Yea, I actually have it mentioned right above the code snippet that the easiest way to use the plugin is via that action.

My bad. Thank you!

**Pull request merged**